version 1.7

EvgeniyKuch opened the pull request:
##### Приложение, хранящее информацию о книгах в библиотеке (на MongoDB).
Подключён Spring Boot Actuator.
Включены метрики, healthchecks и logfile.
Реализован свой собственный HealthCheck индикатор (на основе наличия книг)

Commit:
version 1.7

##### Приложение, хранящее информацию о книгах в библиотеке (на MongoDB)
Подключён Spring Boot Actuator.
Включены метрики, healthchecks и logfile.
Реализован свой собственный HealthCheck индикатор (на основе наличия книг)

kataus commented on library-spring-data-mongo-mvc-auth-url/src/main/java/ru/otus/library/actuators/EmptyBaseOfBooksHealthIndicator.java:
```diff
@@ -0,0 +1,31 @@
+    @Override
+    public Health health() {
+        long bookCount = bookRepository.count();
+        if (bookCount != 0) {
```
Это в первую очередь техническая места, а не бизнесовая. Удаление книг не повод отправлять под в перезагрузку.

kataus commented on library-spring-data-mongo-mvc-auth-url/src/main/resources/logback-spring.xml:
```diff
@@ -0,0 +1,47 @@
+        </layout>
+    </appender>
+
+    <appender name="RollingFile"
```
Сложно, обычно пользуемся application.yml, там удобнее короче и нагляднее. Давайте перенесем

Commit:
version 1.8

##### Приложение, хранящее информацию о книгах в библиотеке (на MongoDB)
Подключён Spring Boot Actuator.
Включены метрики, healthchecks и logfile.

Конфиги logfile перенесены в application.yaml, изменен health-check индикатор.
